### DISCUSSION TITLE
Proton Drive support

### DISCUSSION THREAD

**Discussion by b-risk** [remotely-save/remotely-save#738] [Ideas]
I didn't see Proton listed as a source, I pay for pro.

**Comment by fyears** [comment]
i'd love to. but proton drive doesn't provide any open api so it's quite hard to add support

**Reply by b-risk** [reply]
> i'd love to. but proton drive doesn't provide any open api so it's quite hard to add support

Oh really, I thought (being a company that open sources most of their stuff) they would have done that.
Maybe try take a look at this project I found born out of rclone: https://github.com/StollD/proton-drive

**Comment by Zliit** [comment]
I would also love to have Proton Drive support. Is there any planned advancement on this? Would gladly pay a small donation if this gets up and running.

**Comment by claymorwan** [comment]
Still hoping we get this, it'd be nice since it's a secure clpud service 
I've seen app like celeste that can use proton drive so I doubt this is impossible

**Comment by AnomalyCapital** [comment]
Would love to see this option as well

**Comment by CySec53** [comment]
Hi, same here :)

**Comment by sjmeans** [comment]
Likewise!
